autorest/codegen/models/primitive_schemas.py

```diff
@@ -29,12 +34,15 @@ def _add_optional_and_default_value_template_representation(
     description: Optional[str] = None,
     **kwargs: Any
 ):
+    if default_value_declaration and default_value_declaration != '':
+        representation = default_value_declaration.strip('"')
```

Comment:
![image](https://user-images.githubusercontent.com/70930885/131627822-218efeee-5b18-4207-87b7-4591449ab090.png)

@iscai-msft We have to strip it since `default_value_declaration` has surplus `"`. For example:  `'"Enabled"'`